core/src/main/java/pl/agh/edu/ui/component/table/BaseTable.java

```diff
@@ -19,15 +19,16 @@
 import pl.agh.edu.ui.GameSkin;
 import pl.agh.edu.ui.component.CustomScrollPane;
 import pl.agh.edu.ui.component.label.LanguageLabel;
-import pl.agh.edu.ui.utils.FontType;
+import pl.agh.edu.ui.utils.SkinFont;
 import pl.agh.edu.ui.utils.wrapper.WrapperTable;
+import pl.agh.edu.utils.LanguageString;
 
 public abstract class BaseTable extends WrapperTable {
 	protected Table contentRows = new Table();
 	protected Skin skin = GameSkin.getInstance();
 	protected ScrollPane scrollPane = new CustomScrollPane(contentRows, skin, "transparent");
 
-	public BaseTable(List<String> columnNames) {
+	public BaseTable(List<LanguageString> columnNames) {
 		super();
 		innerTable.align(left);
```

Comment:
```suggestion
		innerTable.align(topLeft);
```
jeśli chcemy żeby header był na swoim miejscu przy usuwaniu rzędów

Reply:
spoko ale trochę nie ten scope tego RP, jak co można zrobić drugie z fixem specjalnie na to